doc/users/colormapnorms.rst

```diff
@@ -0,0 +1,132 @@
+colormaps in a non-linear fashion. 
+
+Logarithmic
+---------------------------------
+
+One of the most common transformations is to plot data by taking its
+logarithm (to the base-10).  This transofrmation is useful when there
+are changes across disparate scales that we still want to be able to
+see.  Using :func:`colors.LogNorm` normalizes the data by
+:math:`log_{10}`.  In the example below, there are two bumps, one much
+smaller than the other. Using :func:`colors.LogNorm`, the shape and
+location of each bump can clearly be seen:
+
+.. plot:: users/plotting/examples/colormap_normalizations_lognorm.py
+   :include-source:
+
+Symetric logarithmic
+---------------------------------
+
+Similarly, it sometimes happens that there is data that is positive
+and negative, but we would still like a logarithmic scaling applied to
+both.  In this case, the negative numbers are also scaled
+logarithmically, and mapped to small numbers.  i.e. If `vmin=-vmax`,
+then they the negative numbers are mapped from 0 to 0.5 and the
+positive from 0.5 to 1.
+
+Since the values close to zero tend toward infinity, there is a need
+to have a range around zero that is linear.  The parameter *linthresh*
+allows the user to specify the size of this range (-*linthresh*,
+*linthresh*).  The size of this range in the colormap is set by
+*linscale*.  When *linscale* == 1.0 (the default), the space used for
+the positive and negative halves of the linear range will be equal to
+one decade in the logarithmic range.
+
+.. plot:: users/plotting/examples/colormap_normalizations_symlognorm.py
+   :include-source:
+
+Power-law
+---------------------------------
+
+Sometimes it is useful to remap the colors onto a power-law
+relationship (i.e. :math:`y=x^{\gamma}`, where :math:`\gamma` is the
+power).  For this we use the :func:`colors.PowerNorm`.  It takes as an
+argument *gamma* ( *gamma* == 1.0 will just yield the defalut linear
+normalization):
+
+.. note::
+  
+   There should probably be a good reason for plotting the data using
+   this type of transformation.  Technical viewers are used to linear
+   and logarithmic axes and data transformations.  Power laws are less
+   common, and viewers should explictly be made aware that they have
+   been used.
+
+
+.. plot:: users/plotting/examples/colormap_normalizations_power.py
+   :include-source:
+
+
+Custom normalization: Two linear ranges
+-----------------------------------------
+
+It is possible to define your own normalization.  This example
+plots the same data as the :func:`colors:SymLogNorm` example, but
+a different linear map is used for the negative data values than
+the positive.  (Note that this example is simple, and does not account
+for the edge cases like masked data or invalid values of *vmin* and
+*vmax*)
+
+.. note::
+   This may appear soon as :func:`colors.OffsetNorm`
+
+   As above, non-symetric mapping of data to color is non-standard
+   practice, and should only be used advisedly.  
```

Comment:
'practice for quantitative data and should be used with caution'.  

Think of a map of the world, there is a sharp discontinuity in the color map at sea level.